source/audio.py

```diff
@@ -0,0 +1,210 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2015-2021 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+import atexit
+from comInterfaces.coreAudio.constants import (
+	CLSID_MMDeviceEnumerator,
+	EDataFlow,
+	ERole,
+)
+import comInterfaces.coreAudio.audioclient as audioclient
+import comInterfaces.coreAudio.audiopolicy as audiopolicy
+import comInterfaces.coreAudio.mmdeviceapi as mmdeviceapi
+import comtypes
+import config
+from enum import IntEnum, unique
+import globalVars
+from logHandler import log
+import nvwave
+from typing import Tuple, Optional, Dict, List, Callable, NoReturn

```

Comment:
Most of these should use the standard `list` and `tuple` which are now generic, or types from `collections.abc` which are preferred over stuff from `typing`. For `Optional` you can also use `foo | None` notation, though that is probably more subjective.